Build/Chakra.Build.Clang.props

```diff
@@ -61,7 +61,9 @@
         -Wno-microsoft-extra-qualification
         -Wno-microsoft-default-arg-redefinition
         -Wno-microsoft-exception-spec
-        -v
+	-Wno-clang-cl-pch
```

Comment:
nit: spacing here?

Reply:
Yep, tab check caught this too, looks like my vim instance when run inside the vs dev tools console didn't pick up the default tab configuration...